discord/interactions.py

```diff
@@ -503,6 +504,44 @@ async def delete_original_response(self) -> None:
             proxy_auth=http.proxy_auth,
         )
 
+    async def translate(
+        self, string: Union[str, locale_str], *, locale: Locale = MISSING, data: Optional[Any] = MISSING
```

Comment:
```suggestion
        self, string: Union[str, locale_str], *, locale: Locale = MISSING, data: Any = MISSING
```
None is seemly never used as the type of data